pkg/core/tekton/builder/event_listener.go

```diff
@@ -0,0 +1,44 @@
+package builder
+
+import (
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EventListenerBuilder holds the tekton event listener definition
+type EventListenerBuilder struct {
+	EventListener v1alpha1.EventListener
+}
+
+// NewEventListenerBuilder creates a EventListener with default values.
+func NewEventListenerBuilder(name, namespace string) *EventListenerBuilder {
+	b := &EventListenerBuilder{}
+	b.EventListener = v1alpha1.EventListener{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+	}
+	return b
+}
+
+// ServiceAccount sets a ServiceAccountName to the EventListener spec.
+func (b *EventListenerBuilder) ServiceAccount(name string) {
```

Comment:
Any reason not to do it already in the "constructor"?

Reply:
The idea is to have the minimum required on the constructor so it can be reused in different places/implementations.